code/datums/traits/negative.dm

```diff
@@ -579,3 +579,43 @@
 	lose_text = "<span class='notice'>Your mind finally feels calm.</span>"
 	medical_record_text = "Patient's mind is in a vulnerable state, and cannot recover from traumatic events."
 	hardcore_value = 9
+
+/datum/quirk/allergic
+	name = "Extreme Medicine Allergy"
+	desc = "Ever since you were a kid you always were allergic to certain chemicals..."
```

Comment:
not strictly a bad bit of grammar, but I'd have this read as something more along the lines of "Ever since you were a kid, you've always been allergic to certain chemicals..."

Reply:
Now it's time for YOUR daily grammar check:

You have two redundant temporal modifiers.  Remove the word "always"

"Ever since you were a kid, you've been allergic to certain chemicals..."

Reply:
redundant temporal modifiers? the fuck are those

Reply:
i am banning all grammar posters